src/config.rs

```diff
@@ -173,14 +186,15 @@ pub struct BuilderConfig {
 
 impl BuilderConfig {
     /// Connect to the Builder signer.
-    pub async fn connect_builder_signer(&self) -> Result<LocalOrAws, SignerError> {
+    pub async fn connect_builder_signer(&self) -> eyre::Result<LocalOrAws> {
```

Comment:
why is this type changing?

Reply:
To make it work with the error type `try_join` expects. The other `connect_*` functions return `eyre::Result` types, so it must match the others' return signatures.